lib/flux/app-state.js

```diff
@@ -238,6 +238,10 @@ var actionMap = new ActionMap( {
 						dispatch( this.action( 'selectAllNotes' ) );
 					}
 
+					if ( settings.markdownEnabled ) {
+						dispatch( this.action( 'setEditorMode', { mode: 'edit' } ) );
+					}
```

Comment:
@copons do we need the `if`? should this not be a safe operation to dispatch on every call?

it is not problematic the way it's written. in fact, it is faster this way, though I'm sure this performance impact will never matter.

I like exploring always-safe process and then removing branching where possible